### PR TITLE
CE

### PR DESCRIPTION
**Pull request by LizUpton**
General tidy, style, spelling, spot of rewording, removed note about the specific Digikey part number of the boards' header.

### CODE REVIEW

File: documentation/asciidoc/accessories/iqaudio/advanced.adoc [hunk 1]
```diff
@@ -1,36 +1,35 @@
 == Hardware information
 
-=== GPIO usage / Pin headers
+=== GPIO usage 
 
-Raspberry Pi Audio boards take advantage of a number of Raspberry Pi Pi signals in
-order to operate successfully. Some of these signals are solely for the use of the board and
+Raspberry Pi audio boards take advantage of a number of pins on the GPIO expander in
```
Comment (lurch) [inline documentation/asciidoc/accessories/iqaudio/advanced.adoc:5]: GPIO expander -> GPIO header ?

File: documentation/asciidoc/accessories/iqaudio/advanced.adoc [hunk 2]
```diff
@@ -1,36 +1,35 @@
...
 some can be shared with other peripherals, sensors, etc.
 
 The following Raspberry Pi GPIO pins will be used by the audio boards:
 
-* ALL POWER PINS
-* ALL GROUND PINS
+* All power pins
+* All ground pins
 * GPIO 2/3 (I2C)
 * GPIO 18/19/20/21 (I2S)
```
Comment (lurch) [inline documentation/asciidoc/accessories/iqaudio/advanced.adoc:14]: Presumably the EEPROM-ID pins are also used? GPIO 0 and 1.

File: documentation/asciidoc/accessories/iqaudio/advanced.adoc (hunk 1 above)
Comment (andrum99) [inline documentation/asciidoc/accessories/iqaudio/advanced.adoc:5]: You've got other pending CE's which have `Raspberry Pi Audio Boards` all capitalised - see #2639 and #2641.

`GPIO expander` is a type of chip. (Although [this](https://www.raspberrypi.com/news/gpio-expander/) confused things by reusing the term for a piece of software). The concensus thus far in the repo tends towards `GPIO header` as the term used to describe the 40 pin header on Raspberry Pi boards.

Comment (LizUpton) [reply documentation/asciidoc/accessories/iqaudio/advanced.adoc]: Fixed

Comment (LizUpton) [reply documentation/asciidoc/accessories/iqaudio/advanced.adoc]: Won't fix

File: documentation/asciidoc/accessories/iqaudio/advanced.adoc (hunk 2 above)
Comment (LizUpton) [reply documentation/asciidoc/accessories/iqaudio/advanced.adoc:14]: That'll need some input from one of the engineers - leaving for now.

**Pull request merged**